netlify.toml

```
@@ -3,877 +3,789 @@
   publish = "build/"
 
 [[redirects]]
-  from = "/user-guide/configuration"
-  to = "/user-guide/configure"
+  from = "/user-guide/rules/list
+  to = "/user-guide/rules
```

Are quotes missing (also on other lines)?

```suggestion
  from = "/user-guide/rules/list"
  to = "/user-guide/rules"
```

---

Good catch!. Fixed.